i18n/en-US.json

```diff
@@ -30,20 +30,20 @@
   "REPLUGGED_LINK_NOW": "Link it now",
   "REPLUGGED_PASSPHRASE": "Passphrase",
   "REPLUGGED_PLUGINS": "Plugins",
-  "REPLUGGED_PLUGINS_DISABLE": "Disable plugins",
+  "REPLUGGED_PLUGINS_DISABLE": "Disable Plugins",
   "REPLUGGED_PLUGINS_DISABLE_NOTE": "This will disable the following plugins:",
-  "REPLUGGED_PLUGINS_ENABLE": "Enable plugins",
+  "REPLUGGED_PLUGINS_ENABLE": "Enable Plugins",
   "REPLUGGED_PLUGINS_ENABLE_NOTE": "This will enable the following plugins:",
   "REPLUGGED_PLUGINS_EXPLORE": "Explore Plugin Store",
   "REPLUGGED_PLUGINS_INSTALLED": "Installed Plugins",
   "REPLUGGED_PLUGINS_LICENSE": "License",
-  "REPLUGGED_PLUGINS_LOAD_MISSING": "Load missing plugins",
-  "REPLUGGED_PLUGINS_OPEN_FOLDER": "Open plugins folder",
+  "REPLUGGED_PLUGINS_LOAD_MISSING": "Load missing Plugins",
```

Comment:
capitalizing plugins here seems odd

Reply:
I’m ambivalent about capitalizing “plugins” here (Discord seems to use title case for context menu items so maybe it should be capitalized), but either way: the capitalization should be consistent within a string. If “plugins” is capitalized, then “missing” and “folder” should also be capitalized.

Reply:
yeah i forgot to do that here, i thought i had :skull: